### DISCUSSION TITLE
Echec de lecture Nrj12 sur Kodi

### DISCUSSION THREAD

**Discussion by Cecilebub** [community/community#63416] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Depuis hier impossible de lire Nrj12 sur Kodi.
Message :
Echec de lecture - Echec de lecture d'un ou de plusieurs fichiers - Consulter le journal pour plus d'informations. 

J'ai consulté le journal il n'y a aucune information. 
J'ai réinstallé depuis le dépôt Catch-up-TV-and-More, rien n'a changé. 

Est-ce que quelqu'un a une solution ?

Merci d'avance.

**Comment by queenofcorgis** [comment]
Hi @Cecilebub,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Kodi. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Reply by Cecilebub** [reply]
I am sorry I don't understand. I had a problem on Kodi a few weeks ago, I posted on your site and had an issue !!
